resources/k8s-artifacts/choreo-connect/rate-limiter-deployment.yaml

```diff
@@ -0,0 +1,120 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2023, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+# Deployment for rate-limiter
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rate-limiter-deployment
```

Comment:
Shall we name this "choreo-connect-rate-limiter" to make it consistent with other components